src/app/core/services/node.ts

```diff
@@ -97,6 +97,7 @@ export class NodeService {
       data: {
         initialClusterData: cluster,
         initialNodeData: {
+          operatingSystemProfile: md.annotations['k8c.io/operating-system-profile'],
```

Comment:
can we define constant for `k8c.io/operating-system-profile`? in common  place so that can be used in both file.